autoscheduler/scraper/management/commands/scrape_depts.py

```diff
@@ -1,28 +1,36 @@
 # Called to execute the beginning of scaping courses
 import time
 import datetime
+from typing import List
 
 from django.core.management import base
 from scraper.banner_requests import BannerRequests
 from scraper.models import Department
+from scraper.management.commands.utils.scraper_utils import get_all_terms
 
-def parse_departments(json, term):
+def parse_departments(json, term) -> List[Department]:
     """ Takes json list of departments and saves them as Department objects """
 
+    depts = []
```

Comment:
Can't suggest since there are removed lines but you can replace this function with
```python
    return [Department(id=dept["code"]+term, code=dept["code"],
                       description=dept["description"], term=term)
            for dept in json]
```